### DISCUSSION TITLE
pls check network tab easier and mimick [Aug 02]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3060] [Discord Threads]
> 💁 **o1lab**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Nils**  
#### What exactly do you mean with network tab?   
####    
#### I'm writing a script to migrate data from Highrise to NocoDB. For this I want also to create the tables with script. Therefore, I need an API which allows me to add create the tables.  
#  
> 💁 **o1lab**  
#### I meant, you can open the network table in browser console.   
#### then create a simple table on nocodb   
#### it will show what fields we send to create timestamps  
#  
> 💁 **o1lab**  
#### looking up highrise  
#  
> 💁 **o1lab**  
#### its a CRM  
#  
> 🙋‍♂️ **Nils**  
#### Ah, thanks!  
#  
> 🙋‍♂️ **Nils**  
#### Yeah Highrise is a CRM. The company that I'm working for wants to use NocoDB as their CRM  
#  
> 🙋‍♂️ **Nils**  
#### With +50.000 contacts  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1003587303064748042)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
